apps/docs/src/app/schema/pipes/type-casting.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { AbstractControl, FormControl, FormGroup } from '@angular/forms';
+
+@Pipe({ name: 'asFormGroup' })
+export class AsFormGroupPipe implements PipeTransform {
```

Comment:
Wasn't really considering this as something public, because I didn't meet enough use cases for this throughout the app. As for now, placing this abstract pipe in the same file, we can move it to our cdk once it's available in `main` branch. Btw, it's a part of another discussion, but we should not have cdk inside `fn` package to leverage it's stuff in core and platform